packages/babel-types/package.json

```diff
@@ -30,7 +30,9 @@
   "devDependencies": {
     "@babel/generator": "workspace:*",
     "@babel/parser": "workspace:*",
-    "chalk": "^4.1.0"
+    "chalk": "^4.1.0",
+    "fs-extra": "^10.0.0",
```

Comment:
I guess we can replace `fs-extra` by `JSON.parse(fs.readFileSync(#))`?

Reply:
I just switched it to `promisify(fs.readFile)`, wouldn't want to block the entire process

Reply:
argh, didn't realize tests run on node 8 though, i'll do it callback style